upgrading.md

```diff
@@ -17,7 +17,15 @@ command from the root of your app (the same directory that contains the
 $ flutter upgrade
 ```
 
+If you've modified your `pubspec.yaml` file, or you want to update
```

Comment:
Maybe add an H2 (`##`) header here, to denote the start of a new section. e.g. "Upgrading your packages"